app/controllers/help_controller.rb

```diff
@@ -10,12 +10,6 @@ def tour
     render locals: { full_width: true }
   end
 
-  def ab_testing
```

Comment:
This page can stay, it's a permanent example of an A/B test used by the [smoke tests][] to verify that we've configured everything correctly:

https://www.gov.uk/help/ab-testing

More context in https://github.com/alphagov/frontend/pull/1099.

Probably also means we can keep the gem in the gemfile and keep some more tests.

[smoke tests]: https://github.com/alphagov/smokey/blob/master/features/ab_testing.feature